topics/browser-extension/index.md

```diff
@@ -0,0 +1,7 @@
+---
+display_name: Browser extension
+short_description: A browser extension is a small software module for customizing a web browser.
+topic: browser-extension
+related: javascript, typescript
+---
+A browser does typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.
```

Comment:
How about this? 🤔 

```suggestion
Web browsers typically allow a variety of extensions, including user interface modifications, cookie management, ad blocking, and the custom scripting and styling of web pages.
```

(Apologies for the back-and-forth. Getting things worded clearly is tough!)